tests/integrations/docker/test_Docker__run.py

```diff
@@ -100,12 +100,12 @@ def test_simple_verbose_call(mock_docker, tmp_path, capsys):
             'world',
         ]
     )
-    assert capsys.readouterr().out == (
-        ">>> docker run --tty "
+    assert capsys.readouterr().out.splitlines()[-1] == (
+        ">>>     docker run --tty "
```

Comment:
What's the origin of this extra space?

Reply:
Updated this test to check entire output.

The extra spaces are there because command being executed is indented now.

This makes more sense in the context of all debug output; for instance:

```
❯ briefcase create -vv
>>> 
>>> Running Command:
>>>     docker --version
>>> Environment:
>>>     LESSCLOSE=/usr/bin/lesspipe %s %s
[OMITTED]
>>>     DISPLAY=:0
>>> Command Output:
>>>     Docker version 20.10.14, build a224086
>>> Return code: 0
[TRUNCATED]
```

Reply:
Ah - that makes sense. Thanks.